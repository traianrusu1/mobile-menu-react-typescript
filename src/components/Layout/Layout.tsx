import React from 'react';
import styles from './Layout.module.css';
import { Toolbar } from '..';

interface Props {
  children: React.ReactNode;
}

const Layout: React.FC<Props> = ({ children }: Props) => {
  return (
    <div className={styles.layout}>
      <Toolbar myProp="" />
      {children}
    </div>
  );
};

export default Layout;
