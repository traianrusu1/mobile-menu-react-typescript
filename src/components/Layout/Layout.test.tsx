import React from 'react';
import { render, cleanup } from '@testing-library/react';

import Layout from './Layout';

afterEach(cleanup);

describe('<Layout />', () => {
  test('renders', () => {
    const { container } = render(<Layout> Test Children </Layout>);
    expect(container).toMatchSnapshot();
  });
});
