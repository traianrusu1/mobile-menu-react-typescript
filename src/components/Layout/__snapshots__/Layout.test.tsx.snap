// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Layout /> renders 1`] = `
<div>
  <div
    class="layout"
  >
    <div
      class="toolbar"
    >
      Toolbar
      
    </div>
     Test Children 
  </div>
</div>
`;
